src/linux/ConfigurationXml.S

```diff
@@ -5,6 +5,8 @@ configurationXmlStart:
     .incbin "../../resources/data/configuration.xml"
 configurationXmlEnd:
     .global configurationXmlNullTerminator
+#ifndef __arm__
```

Comment:
We will need this to be null terminated somehow otherwise the read will fail and surge can crash in some situations.